contracts/contracts/coordination/StakeInfo.sol

```diff
@@ -13,17 +13,19 @@ import "../../threshold/IAccessControlApplication.sol";
 contract StakeInfo is AccessControl, IUpdatableStakes, IAccessControlApplication {
 
     bytes32 public constant UPDATE_ROLE = keccak256("UPDATE_ROLE");
+    address public polygonChild;
 
     struct Stake {
         address operator;
         uint96 amount;
         // TODO: what about undelegations etc?
     }
 
-    constructor(address[] memory updaters){
+    constructor(address _polygonChild, address[] memory updaters){
         for(uint i = 0; i < updaters.length; i++){
             _grantRole(UPDATE_ROLE, updaters[i]);
         }
+        polygonChild = _polygonChild;
```

Comment:
The polygon child is an updater, so there's no need to track it. An access-control based design allows the StakeInfo contract to be completely unaware of how the update mechanism works: it could be a cross-chain messaging contract, but also could be a multisig or a privileged EOA.